edge-helm-charts/charts/mqtt-broker/templates/service.yaml

```diff
@@ -0,0 +1,18 @@
+{{- $k8sname := include "acs.k8sname" . }}
+apiVersion: v1
+kind: Service
```

Comment:
Does this default to `NodePort` type?

Reply:
ClusterIP. The external access is provided by `externalIPs` and doesn't need a NodePort. A NodePort would be restricted to a stupid port number; with `externalIPs` we just expose the Service port (1883) directly on an external IP.